vagrant/k8s/Vagrantfile

```diff
@@ -114,133 +105,121 @@ Vagrant.configure(2) do |config|
+                       ip: "50.1.1.200",
+                       virtualbox__intnet: "contiv_yellow"
 
-        quagga2.vm.provision "shell" do |s|
-          s.inline = provision_quagga
-          s.args = [http_proxy, https_proxy]
-        end
+      quagga2.vm.provision "shell" do |s|
+        s.inline = provision_quagga
+        s.args = [http_proxy, https_proxy]
       end
-   end
+    end
+  end
 
-  #config.ssh.password = 'vagrant'
   config.ssh.insert_key = false
   config.ssh.private_key_path = "./../../scripts/insecure_private_key"
   
   config.vm.synced_folder "./export", "/shared"
+  config.vm.synced_folder "./../../../../../","/opt/gopath/src"
+  config.vm.synced_folder "./../../bin","/opt/gopath/bin"
+
+  n = 0
   cluster.each do |role, member_list|
     member_list.each do |member_info|
       config.vm.define vm_name = member_info["name"] do |c|
-         if ENV['CONTIV_K8'] == "1" then
-       	   c.vm.box_version = "0.0.4"
+         if ENV['CONTIV_K8S_LEGACY'] == "1" then
+           c.vm.box_version = "0.0.4"
            box_name = vm_name
-	    if  role == "master" then
-	      c.vm.box = "contiv/k8master"
-              c.vm.synced_folder "./../../","/opt/gopath/src/github.com/contiv/netplugin"
-	      c.vm.provision "shell" do |s|
-		s.inline = provision_k8
-                s.args = [ENV["http_proxy"] || "", ENV["https_proxy"] || ""]
-              end
-	      c.vm.provision "shell" do |s|
-                s.inline = provision_master
-	        s.args = [ENV["http_proxy"] || "", ENV["https_proxy"] || ""]
-              end
-              c.vm.network "forwarded_port", guest: 9999, host: 9999
-       	   else
-	      if vm_name =="k8node-01" then
- 	      	c.vm.box = "contiv/k8node01"
- 	      else
- 		 if vm_name == "k8node-02" then
- 		    c.vm.box = "contiv/k8node02"
-                  else
-		    c.vm.box = "contiv/k8node03"
- 		 end
- 	      end	
-	   end
-	   c.vm.synced_folder "./../../bin","/opt/gopath/bin"
-	   c.vm.provision "shell" do |s|
-              s.inline = provision_k8
-               s.args = [ENV["http_proxy"] || "", ENV["https_proxy"] || ""]
-           end
-	else
-          if ENV['CONTIV_NODE_OS'] && ENV['CONTIV_NODE_OS'] == "rhel7" then
+           case vm_name
+	         when "k8master"
+               c.vm.box = "contiv/k8master"
+             when "k8node-01"
+               c.vm.box = "contiv/k8node01"
+             when "k8node-02"
+               c.vm.box = "contiv/k8node02"
+             when "k8node-03"
+               c.vm.box = "contiv/k8node03"
+             end
+	     else
+          if ENV['CONTIV_NODE_OS'] == "rhel7" then
             # Download rhel7.2 box from https://access.redhat.com/downloads/content/293/ver=2/rhel---7/2.0.0/x86_64/product-software
             # Add it as rhel7 vagrant box add rhel-cdk-kubernetes-7.2-29.x86_64.vagrant-virtualbox.box --name=rhel7
             c.vm.box = "rhel7"
           else
             c.vm.box = "contiv/k8s-centos"
```

Comment:
Do we really need to use this box or can we use another one? Is there something specific packaged in this Vagrant box?